src/client/pfs_file.go

```diff
@@ -380,6 +380,23 @@ func (ctfsc *CreateFileSetClient) Close() (*pfs.CreateFileSetResponse, error) {
 	return ret, nil
 }
 
+// GetFileSet gets a file set for a commit.
+func (c APIClient) GetFileSet(repo, branch, commit string) (_ string, retErr error) {
+	defer func() {
+		retErr = grpcutil.ScrubGRPC(retErr)
+	}()
```

Comment:
This removes the structured error. I don't know what the benefit is, other than maybe formatting in the CLI.  It shouldn't be in the client, which can be used programmatically.

Reply:
Yeah, this is the common pattern we have been following, but the go client needs a full rewrite and consistent error handling.